docs/static/upgrading.asciidoc

```diff
@@ -52,20 +53,6 @@ This procedure downloads the relevant Logstash binaries directly from Elastic.
 some Logstash plugins have changed in the 5.x release.
 5. Restart your Logstash pipeline after updating your configuration file.
 
-[[upgrading-logstash-pqs]]
-=== Upgrading with Persistent Queues Enabled
```

Comment:
Since you are changing the ID, make sure you open an issue in the website repo to get a redirect set up for any versions impacted by this change.

Reply:
@dedemorton Technically, I didn't change the ID.  I deleted that topic and added a new very specific topic with much more narrow use.  Is the redirect still the right approach?

Reply:
@karenzone I wasn't sure which branches you would be changing when I added this comment. If you delete an ID that's used to set the html page name in the published docs (for example, https://www.elastic.co/guide/en/logstash/6.2/upgrading-logstash-pqs.html) then you should set up redirects.

Reply:
I updated the topic to re-use the old ID.  The content is completely different, but somebody searching for the old info might be interested in the new info.